index.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Bakery Store</title>
+    <link rel="stylesheet" href="https://iceberg-ui.netlify.app/styles.css" />
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <nav class="nav nav-fixed">
+      <div class="nav-section">
+        <span class="nav-title">Bakery Store</span>
+      </div>
+      <div class="nav-section nav-section-right">
+        <button class="btn btn-primary btn-sm btn-icon">
+          <span class="material-icons"> person </span>
+          User
+        </button>
+        <button class="btn btn-primary btn-sm btn-icon">
+          <span class="material-icons"> shopping_cart </span>
+          Cart
+        </button>
+      </div>
+    </nav>
+
+    <main class="nav-fixed-adjust">
+      <div class="landing-img-container">
+        <img
+          class="landing-img"
+          src="https://source.unsplash.com/VnM6_liIRJ0"
+          alt=""
+        />
+      </div>
+      <div class="grid grid-col-4 categories">
+        <a class="card card-overlay" href="/cakes.html">
+          <img
+            class="card-img category-img"
+            src="https://source.unsplash.com/vdx5hPQhXFk"
+          />
+          <div class="overlay-content">
+            <div class="card-title">Cakes</div>
+          </div>
+        </a>
+        <a class="card card-overlay" href="/cakes.html">
+          <img
+            class="card-img category-img"
+            src="https://source.unsplash.com/YDvfndOs4IQ"
+          />
+          <div class="overlay-content">
+            <div class="card-title">Cookies</div>
+          </div>
+        </a>
+        <a class="card card-overlay" href="/cakes.html">
+          <img
+            class="card-img category-img"
+            src="https://source.unsplash.com/qZ6uvJHLHFc"
+          />
+          <div class="overlay-content">
+            <div class="card-title">Doughnuts</div>
```

Comment:
Donuts*?

Reply:
Both are correct, Donut is actually a simplified spelling of Doughnut.